plugins/techdocs-react/report.api.md

```diff
@@ -122,6 +132,13 @@ export type TechDocsReaderPageValue = {
   onReady?: () => void;
 };
 
+// @public
+export type TechDocsRouteFunc = RouteFunc<{
```

Comment:
Maybe don't make this type public, and instead inline it in the `buildTechDocsURL` above? Less API pollution